frontend/JavaScript/category.js

```diff
@@ -0,0 +1,43 @@
+window.onload = function(){
+fetch(`http://127.0.0.1:7002/categoryTree`,{
+    method : 'GET',
+    mode :'cors',
+    headers:{
+      'Access-Control-Allow-Origin':'*',
+      'Accept': 'application/json',
+      'Content-Type': 'application/json;charset=utf-8',
+      
+    }
+    
+  }).then(response => response.json()).then(data =>{
+    cat1 = data[0];
+    cat2 = data[1];
+    cat3 = data[2];
+    var cat_container=document.getElementById("dropdownheader");
+    cat_container.innerHTML =``;
+    for( let i = 0; i < cat1.length; i++){
+      var catId = cat1[i][1];
+      var catValue = cat1[i][0];
+      var value = cat1[i][0];
```

Comment:
Use let instead of var for creating variables.

Reply:
Let has been used instead of var for declaration.